client/coral-framework/services/i18n.js

```diff
@@ -3,18 +3,20 @@ import has from 'lodash/has';
 import get from 'lodash/get';
 import merge from 'lodash/merge';
 
-import esTA from '../../../node_modules/timeago.js/locales/es';
-import frTA from '../../../node_modules/timeago.js/locales/fr';
-import pt_BRTA from '../../../node_modules/timeago.js/locales/pt_BR';
+import esTA from 'timeago.js/locales/ta'
+import frTA from 'timeago.js/locales/ta';
+import pt_BRTA from 'timeago.js/locales/ta';
+import da_DKTA from 'timeago.js/locales/ta';
```

Comment:
Looks like some copy and paste error, they now all end with `ta` ^^

Reply:
Going with the previous pattern this line should be `import daTA from 'timeago.js/locales/da';`